src/App.js

```diff
@@ -1,13 +1,14 @@
 import './App.css';
-import MainPage from './containers/MainPage/MainPage';
+import MainPage from './containers/MainPage/components/MainPage/MainPage';
```

Comment:
Why did you decide to move MainPage from containers to component. It still looks like container to me. But even if we will call all things just components (without using containers at all) it will also be correct. What is weird, that you have container MainPage and inside it you have component MainPage. Why do we leave this container then.
Let's clear this up somehow. If you are not comfortable with containers right now, let's all be just components - smart(which is sometimes called containers, contain a lot of logic and other components) and dumb (super simple component with easy inner logic and presentation, might contain other small components)

Reply:
Oh, i see - you do have container MainPage AND component MainPage. So let just give this component another name than MainPage😀. It contains water counter and navbar, so it might be WaterTracker itself, because it's main component

Reply:
Just don't give components same name, they should be unique (even if component is container)